canonical-websites/www.ubuntu.com
# Remove broken ‘share-this’ social include

## Done

Remove broken ‘share-this’ social include. These have been marked for removal in v1 migration so may as well remove them all at once.

## QA

- Check out this feature branch
- Check code to make sure only the includes have been removed
